tests/test_metadata_fields.py

```diff
@@ -77,3 +77,21 @@ def test_metadata_field_filter_sum(self):
                           {'count': 2, 'field': 'name', 'type': 'text'},
                           {'count': 2, 'field': 'subject', 'type': 'text'},
                           {'count': 2, 'field': 'type', 'type': 'keyword'}], json_result)
+
+    def test_dont_count(self):
```

Comment:
It could be interesting to add a test that the command param `--count `returns the same as without the params. Or maybe the `--count ` is not necessary in the end?